inventory/public/style.css

```diff
@@ -11,13 +11,18 @@ body{
 	margin-right:0;
 }
 
-#inventory {
+#inventory-wrapper {
 	position: absolute;
 	top: 0px;
 	left: 0px;
 	width: 960px;
-	height: 720px;
+	height: 930px;
 	z-index: 2;
+	background: black;
```

Comment:
Is the idea that this blocks the item notifications? Won't it block the background too, when using it in the media component?

Reply:
Oh. Right. Yes. Dammit. I need to find another solution then. Crap.